util/getFullName.js

```diff
@@ -0,0 +1,9 @@
+export default function getFullName(firstName, lastName) {
```

Comment:
To move the object deconstructing logic to the function all we need to do is change  `getFullName(firstName, lastName)`. to `getFullName({ firstName,  lastName })`